include/univalue.h

```diff
@@ -177,76 +176,9 @@ class UniValue {
     const UniValue& get_array() const;
 
     enum VType type() const { return getType(); }
-    bool push_back(std::pair<std::string,UniValue> pear) {
-        return pushKV(pear.first, pear.second);
-    }
     friend const UniValue& find_value( const UniValue& obj, const std::string& name);
 };
 
-//
-// The following were added for compatibility with json_spirit.
-// Most duplicate other methods, and should be removed.
-//
-static inline std::pair<std::string,UniValue> Pair(const char *cKey, const char *cVal)
```

Comment:
Seems fine to remove in principle, but aren't we using these currently?
https://github.com/bitcoin/bitcoin/blob/cdd6bbf10a818d243f973f628e465cc1df98691a/src/rpc/blockchain.cpp#L79-L106? There and in other places.

Reply:
Indeed. That code is updated to no longer use `push_back(Pair(...` in https://github.com/bitcoin/bitcoin/pull/11386 (which is what prompted PR #5)

Reply:
It looks like we might need a replacement for bitcoin/bitcoin#11386, since @karel-3d didn't have enough time to maintain https://github.com/bitcoin-core/univalue/pull/5#issuecomment-352839011.

Reply:
Also it looks like the dependency between bitcoin/bitcoin#11386 is currently circular. 11386 depends on the "Pushing boolean value to univalue correctly" commit in this PR. And the "Remove deprecated std::pair wrappers" commit in this PR depends on 11386. Maybe should drop the "Remove deprecated std::pair wrappers" commit here this PR can be merged, and then the pushKV changes in 11386 can then be merged, and then the Pair code can be removed later.

Reply:
Can we do the following:

1. merge this PR into bitcoin-core/univalue
2. Add two new subtree merge commits to https://github.com/bitcoin/bitcoin/pull/11386. One that takes the first commit from this PR (as the first commit in 11386), and one that takes the remaining commits from this PR (as the final commits in 11386)

Reply:
> Can we do the following

It wouldn't be my preference to partially merge pieces of a univalue PR in a subtree bump PR, because it would require dropping PR metadata (see https://github.com/bitcoin/bitcoin/pull/11952#issuecomment-352887560), but you could do this. I'd prefer to just deprecate more gently and drop the "Remove deprecated" commit here.